.gitignore

```diff
@@ -0,0 +1,163 @@
+data/*.csv
```

Comment:
There's a small change: `data/*.csv` are created by `gemelnet.py` (added at this PR).